docs/configuration.md

```diff
@@ -152,4 +152,39 @@ level = "info"        # "debug", "info", "warning"
 # some functions took to execute
 performance_log = false
 performance_log_directory = "performance_logs"
-```
\ No newline at end of file
+```
+
+
+## Environment variables
+
+| Environment Variable  | config section         | config file entry          |
+| --------------------- | ---------------------- | -------------------------- |
+|                       | main                   | backend                    |
+|                       | main                   | import_ips                 |
+|                       | main                   | import_prefixes            |
+|                       | main                   | import_intf_status         |
+|                       | main                   | import_vlans               |
+|                       | main                   | import_cabling             |
+|                       | main                   | excluded_platforms_cabling | 
+|                       | main                   | nbr_workers                |
+|                       | main                   | configs_directory          |
+| BATFISH_ADDRESS       | batfish                | address                    |
+| BATFISH_NETWORK_NAME  | batfish                | network_name               |
+| BATFISH_SNAPSHOT_NAME | batfish                | snapshot_name              |
+| BATFISH_API_KEY       | batfish                | api_key                    |
+| BATFISH_PORT_V1       | batfish                | port_v1                    |
+| BATFISH_PORT_V2       | batfish                | port_v2                    |
+| BATFISH_USE_SSL       | batfish                | use_ssl                    |
+|                       | drivers.mapping        | <should this be made into ENV variables?> |
+|                       | inventory              | supported_platforms        |
+|                       | inventory              | backend (duplicate of main.backend?) |
```

Comment:
Yes the doc is not correct, this setting doesn't exist.
please could you remove it from this table and update the doc as well ?